### ISSUE TITLE
Table 'glpi.glpi_plugin_resources_contracttypes doesn't exist

### ISSUE THREAD

**Issue by Krisstof** [InfotelGLPI/resources#47]
Hi
- Plugin version : 2.4.1
- GLPI Version : 9.2
When installing the plugins, message : "Table 'glpi.glpi_plugin_resources_contracttypes'" doesn't exist.
Could you give me "column_name" and "column_type" in order to manually create the missing table in db.

Best

**Comment by mholttech** [comment]
I'm having the same issue.  Here are the logs from my fress hinstall of GLPI 9.2

cat php-errors.log
````
Test
Test
2017-11-16 16:33:39 [@glpi]
  *** PHP Notice(8): Undefined index: glpiactiveprofile
  Backtrace :
  inc/html.class.php:1198
  inc/html.class.php:1769                            Html::includeHeader()
  front/login.php:83                                 Html::nullHeader()
````

cat sql-errors.log
````2017-11-16 16:34:09 [2@glpi]
  *** MySQL query error:
  SQL: INSERT INTO `glpi_plugin_resources_contracttypes` ( `id`, `name`,`comment`)
         VALUES (1, 'Long term contract', '')
  Error: Table 'glpidb.glpi_plugin_resources_contracttypes' doesn't exist
  Backtrace :
  plugins/resources/hook.php:55
  inc/plugin.class.php:730                           plugin_resources_install()
  front/plugin.form.php:48                           Plugin->install()
````

**Comment by amma35** [comment]
For the version GLPI 9.2, it is normal the plugin is under development and therefore not functional for the moment. Installing a new version is not yet implemented

**Comment by amma35** [comment]
Fix in [2.4.0](https://github.com/InfotelGLPI/resources/releases/tag/2.4.0)
